index.html

```diff
@@ -0,0 +1,163 @@
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Foodie Hunter</title>
+    <link href="./assets/css/style.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+        integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <script rel="javascript" src="./assets/js/index.js"></script>
+    <link href="./assets/css/nouislider.css" rel="stylesheet">
+    <script src="./assets/js/nouislider.js"></script>
+
+</head>
+
+<body>
+    <!-- Header -->
+    <header>
+        <div class="top-menu">
+            <div class="menu-item">
+                <a href="index.html">
+                    <img src="./assets/images/foodielogo.png" alt="Logo">
+                </a>
+            </div>
+            <div class="main-pages menu-item">
+                <a href="#">Home</a>
+                <span class="small-circle"></span>
+
+                <a href="#">About</a>
+                <span class="small-circle"></span>
+
+                <a href="#">Services</a>
+                <span class="small-circle"></span>
+
+                <a href="#">Blog</a>
+                <span class="small-circle"></span>
+
+                <a href="#">Contacts</a>
+            </div>
+        </div>
+        <div class="description">
+            <h1 class="title-text">Foodie Hunter</h1>
+            <hr class="title-line" />
+            <p class="heading-text">Good food is the foundation of genuine happiness.</p>
+        </div>
+    </header>
+
+
+    <!-- Main Site -->
+    <main id="site-main">
+        <div class="bar">
+            <div class="subnav-bar">
+                <div class="subnav-menu">
+                    Category
+                    <i class="fa-solid fa-angles-right icon"></i>
+                    <div class="subnav-content">
+                        <p> Entrees</p>
+                        <p> Salads</p>
+                        <p> Second Course</p>
+                    </div>
+                </div>
+                <div class="subnav-menu">
+                    Sorting
+                    <i class="fa-solid fa-angles-right icon"></i>
+                    <div class="subnav-content">
+                        <p> Alphabetical</p>
+                        <p> Price Ascending</p>
+                        <p> Price Descending</p>
+                    </div>
+                </div>
+                <div>
+                    Vegetarian
+                    <i class="fa-solid fa-angles-right icon"></i>
+                </div>
+            </div>
+
+            <div class="price-filter subnav-bar">
+                <p> Price: </p>
+                <p id="min-slider"></p>
+                <div id="uislider"></div>
+                <p id="max-slider"></p>
+            </div>
+
+
+        </div>
+
```

Comment:
You have some leftover newlines, between closing divs
```
  </div>


</div>
```

and here, you don't need 4 newlines to improve readability, you almost never need more than 1. these just make the file longer

Reply:
It is fine to have things like:
```
<div1>
  <content>
</div1>

<div2>
  <content2>
</div2>
```

Here, between actual "components" or "blocks", it does improve readability. But if we have:
```
<div>
  <div>
    <content>
  </div>
--- no newline here
</div>
```

it's better without the newline